client/view/base/BrowsersStat.css

```diff
@@ -0,0 +1,126 @@
+.BrowsersStat__globalCoverageBar {
+  display: flex;
+  gap: 2px;
+  justify-content: flex-start;
+  width: 100%;
+  padding: 0;
+  overflow: hidden;
+  counter-reset: alpha 0;
+  background-color: var(--main-bg);
+  border-radius: 6px;
+}
+
+.BrowsersStat__globalCoverageBarItem {
+  box-sizing: border-box;
+  width: calc(var(--p, 0) * 1%);
+  overflow: hidden;
+  line-height: 32px;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  list-style: none;
+  counter-increment: alpha -1;
+  background-color: var(--accent-alternave);
+  opacity: var(--a, 0%);
+
+  /* TODO types for this variables */
+  --p: 0;
```

Comment:
Didn’t get what a reason for these custom properties. Using better names will be helpful.

Reply:
We still have one-letter properties https://github.com/browserslist/browserl.ist/pull/306/files#diff-cf8d12ca5d25f78b1f7f9696966e2a61fd8c25107e6437159fadf37f720a1184R100